bim2sim/sim_settings.py

```diff
@@ -317,7 +317,7 @@ def check_value(self, bound_simulation_settings, value):
     def __set__(self, bound_simulation_settings, value):
         """This is the set function that sets the value in the simulation setting
         when calling sim_settings.<setting_name> = <value>"""
-        if not isinstance(value, Path):
+        if not isinstance(value, Path) and not value == self.default:
             if value:
```

Comment:
If a path cannot be detected by isinstance(value, path) but is not None by default, this condition is not entered and the value is not checked for being path or not. Shouldn't we apply a two-stage check here?

Reply:
Good point, I changed this.